test/e2e/specs/site-editor/template-part.spec.js

```diff
@@ -0,0 +1,111 @@
+/**
```

Comment:
Let's split this into a different PR to better review/manage it?

Reply:
Good reminder, I'll remove it.

Thanks for all the review feedback and helping get this to a good place.